code/confetti.js

```diff
@@ -0,0 +1,219 @@
+
+// THIS CODE I DONT QUIT UNDERSTAND BUT IT MAKES CONFETTI RAIN :)
+
+

```

Comment:
Love that you use this effect 👯

Reply:
Thanks Jessica! Yeah, really nice effect, but a lot of code that I don't understand :)